v0/destinations/drip/data/DripIdentify.json

```diff
@@ -0,0 +1,120 @@
+[
+    {
+        "destKey": "email",
+        "sourceKeys": ["traits.email","context.traits.email"],
+        "required":false
+    },
+    {
+        "destKey": "new_email",
+        "sourceKeys": ["traits.newEmail", "context.traits.newEmail"],
+        "required": false
+    },
+    {
+        "destKey":"phone",
+        "sourceKeys":["traits.phone","context.traits.phone"],
+        "required":false
+    },
+    {
+        "destKey":"first_name",
+        "sourceKeys":["traits.firstName",
+            "context.traits.firstName",
+            "traits.firstname",
+            "context.traits.firstname"
+        ],
+        "required": false
+    },
+    {
+        "destKey":"last_name",
```

Comment:
`sourceFromGenericMap: true` can be used for keys such as this. These mappinggs are there by default in `GenericFieldMapping.json`